lib/src/builders/json_dropdown_button_form_field_builder.dart

```diff
@@ -280,10 +278,23 @@ class _JsonDropdownButtonFormFieldWidgetState
       );
     }
 
-    final itemEntries = <String, dynamic>{};
-    if (widget.builder.items != null) {
-      final items = <DropdownMenuItem>[];
+    _items = widget.builder.items;
+    _value = widget.builder.value;
+  }
 
+  @override
+  Widget build(BuildContext context) {
+    // If builder values has been changed then the ones that we use for state
+    // also has to be changed
+    if (widget.builder.items != _items || widget.builder.value != _value) {
```

Comment:
A `setState` really shouldn't be called inside of a `build` function.  It's generally preferred to do something like this inside of a `didUpdateWidget` or `didChangeDependencies`

Reply:
Ok - moved to `didUpdateWidget` + done some cleaning